### PR TITLE
chore(css): tabs display as tabs on tablets

### LINKED ISSUE #11532: On mobile/tablet, tabs don't need to be blocks
Particularly for the All/Mine/Friends tabs, these could just stay tabs all the way down to mobile instead of stacking. Can't we just allow all tabs to wrap?

### PR DESCRIPTION
**Pull request by mrclay**
This layout allows tabs to wrap in cases where they must.

Fixes #11532

cc @hypeJunction

### CODE REVIEW

**Comment by hypeJunction** [comment]
As long as overflox-x is scroll, I am ok with this

**Comment by jdalsem** [comment]
this introduces a new breakpoint. please update the theme/style docs if we want this

**Comment by mrclay** [comment]
Is the single breakpoint a principle encouraged somewhere?

**Comment by mrclay** [comment]
Why `overflow-x: scroll`? it results in this nastiness: 
screen shot 2017-12-22 at 12 16 03 pm

**Comment by hypeJunction** [comment]
`overflow: auto` looks ugly in desktop browsers, but it's fine on mobile. Better than weird stacking. Also, it's really bad UX to have so many tabs like we do.

**Comment by mrclay** [comment]
Tabs scrolling left-right introduces the problem of tabs being out-of-sight, and the need to awkwardly swipe/scroll right to access them. What if we: 1. used style that looks OK wrapped to a second line, or 2. could automatically switch to the mobile layout as needed...

**Comment by hypeJunction** [comment]
There is also this: https://github.com/hypeJunction/Elgg-ui_responsive_tabs

**Comment by hypeJunction** [comment]
Then you also have the gallery toggle which shouldn't be a tab.

**Comment by mrclay** [comment]
Are we open to an alternate (wrap-friendly) tab style, like 2px of primary (topbar) color beneath the selected tab? If not, I think I'll just abandon this. The scroll CSS doesn't work with how we position the selected tab so I think there'd always be a scroll bar.

**Comment by hypeJunction** [comment]
I guess this should be ok. Sites can deal with overflowing tab issues if they have them

**Comment by mrclay** [comment]
I nudged this a bit toward our style recommendations. Flex in all media.

File: docs/guides/themes.rst [hunk 1]
```diff
@@ -77,8 +77,8 @@ Theming Principles and Best Practices
 
 	@media screen and (max-width: 820px) {
```
Comment (hypeJunction) [inline docs/guides/themes.rst]: Change to 50rem

**Pull request merged**